msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/TestCase1561087.java

```diff
@@ -0,0 +1,53 @@
+package com.microsoft.identity.client.msal.automationapp.testpass.broker;
+
+import com.microsoft.identity.client.msal.automationapp.R;
+import com.microsoft.identity.labapi.utilities.client.LabQuery;
+import com.microsoft.identity.labapi.utilities.constants.AzureEnvironment;
+import com.microsoft.identity.labapi.utilities.constants.TempUserType;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+// Flight settings
+// https://identitydivision.visualstudio.com/Engineering/_workitems/edit/1561087
+public class TestCase1561087  extends  AbstractMsalBrokerTest {
```

Comment:
Since this test case is specific to BrokerHost do we need `@SupportedBrokers` annotation?